ui/src/components/credentials/CredentialDetails.tsx

```diff
@@ -173,6 +174,11 @@ export function CredentialDetails() {
               className="centered"
               extra={
                 <Space>
+                  <Link target="_blank" to={generatePath(ROUTES.issuedQR.path, { credentialID })}>
+                    <Button icon={<IconQR />} type="link">
+                      Scan
```

Comment:
This needs to be clearer and in line with the credential link, like "View QR code"

Reply:
I guess you mean to add the Link URL also to the details, right? Agreed.

Also, I'd name the button "QR code link" instead of "Scan"

Reply:
I've followed the old design for the Download but ok

Reply:
Fair enough. Do as you prefer.